x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.tsx

```diff
@@ -55,44 +40,10 @@ export const EmptyEngineOverview: React.FC = () => {
           </EuiButton>
         </EuiPageHeaderSection>
       </EuiPageHeader>
-      <EuiPageContent>
-        <EuiPageContentHeader>
-          <EuiTitle>
-            <h2>
-              {i18n.translate('xpack.enterpriseSearch.appSearch.engine.overview.empty.subheading', {
-                defaultMessage: 'Setting up the “{engineName}” engine',
-                values: { engineName },
-              })}
-            </h2>
-          </EuiTitle>
-        </EuiPageContentHeader>
-        <EuiPageContentBody>
-          <EuiText color="subdued">
-            <p>{DOCUMENT_CREATION_DESCRIPTION}</p>
-          </EuiText>
-          <EuiSpacer />
-          {/* TODO: <DocumentCreationButtons /> */}
-        </EuiPageContentBody>
-
-        <EuiPageContentHeader>
-          <EuiTitle>
-            <h3>{DOCUMENT_API_INDEXING_TITLE}</h3>
-          </EuiTitle>
-        </EuiPageContentHeader>
-        <EuiPageContentBody>
-          <EuiText color="subdued">
-            <p>{DOCUMENT_API_INDEXING_DESCRIPTION}</p>
-            <p>
-              {i18n.translate('xpack.enterpriseSearch.appSearch.engine.overview.empty.apiExample', {
-                defaultMessage:
-                  'To see the API in action, you can experiment with the example request below using a command line or a client library.',
-              })}
-            </p>
-          </EuiText>
-          <EuiSpacer />
-          {/* <DocumentApiCodeExample engineName={engineName} apiKey={apiKey} /> */}
-        </EuiPageContentBody>
-      </EuiPageContent>
+      <EuiPageContentBody>
+        <DocumentCreationButtons />
+        <DocumentCreationModal />
+      </EuiPageContentBody>
```

Comment:
I'm a personally super excited by how much cleaner this is now - let me know what y'all think and if you agree!

Reply:
I agree, I think this is much nicer. It's cool that we were able to reuse everything for here and in the modal.